### PR TITLE
feat: Add option to store cache in ramdisk/tmpfs 🔥

### PR DESCRIPTION
**Pull request by oscrx**
Add option to store cache in ramdisk/tmpfs 🔥

### CODE REVIEW

**Comment by oscrx** [comment]
It might even be a good default... But it might mess with some people's memory limits.
Memory increase is not shocking though, but the speeeeeeed is 🚀
afbeelding

File: helm-charts/mend-renovate-ce/values.yaml [hunk 1]
```diff
@@ -192,6 +192,13 @@ cachePersistence:
   ## The name of an existing PVC to use for persistence
   existingClaim: ""
 
+## Cache in Memory
+## ref: https://kubernetes.io/docs/concepts/storage/volumes/#emptydir
+##
```
Comment (nabeelsaabna) [inline helm-charts/mend-renovate-ce/values.yaml:197]:
```suggestion
## cachePersistence.enabled takes precedence, make sure its disabled
##
```

File: helm-charts/mend-renovate-ee/values.yaml [hunk 2]
```diff
@@ -344,6 +344,13 @@ dataPersistence:
   ## The name of an existing PVC to use for persistence
   existingClaim: ""
 
+## Cache in Memory
+## ref: https://kubernetes.io/docs/concepts/storage/volumes/#emptydir
+##
```
Comment (nabeelsaabna) [inline helm-charts/mend-renovate-ee/values.yaml:349]:
```suggestion
## cachePersistence.enabled takes precedence, make sure its disabled
##
```

File: helm-charts/mend-renovate-ce/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mend-renovate-ce
-version: 7.3.0
+version: 7.4.0
```
Comment (nabeelsaabna) [inline helm-charts/mend-renovate-ce/Chart.yaml:3]:
```suggestion
version: 7.5.0
```

File: helm-charts/mend-renovate-ee/Chart.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mend-renovate-enterprise-edition
-version: 1.3.0
+version: 1.4.0
```
Comment (nabeelsaabna) [inline helm-charts/mend-renovate-ee/Chart.yaml:3]:
```suggestion
version: 1.5.0
```

**Review by nabeelsaabna — CHANGES_REQUESTED** [review]
add comments

**Comment by oscrx** [comment]
Sorry, this PR slipped my mind, let me know if I need to squash or rebase something :)

**Review by nabeelsaabna — APPROVED** [review]

**Pull request merged**